### PR TITLE
Add readme according to OpenSRP new template

### PR DESCRIPTION
**Pull request by cafootitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,60 @@
...
+|Service Delivery Support                  |PNC visit schedules - at home and at health facility                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Service Delivery Support                  |Schedules for high priority PNC services                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Client Management            |Record services provided during PNC home visit                                                                                                                                  |opensrp-client-native-form (rules engine)                          |
+                                                                |opensrp-server-web, opensrp-client-core, opensrp-client-native-form|
```
Comment (pld) [inline README.md:40]: this line looks like it's missing the Function and Feature, do know what does are? Or is the whole line extraneous and these should be removed?

**Review by pld — CHANGES_REQUESTED** [review]

**Comment by pld** [comment]
otherwise lgtm

File: README.md (hunk 1 above)
Comment (cafootitt) [reply README.md:40]: Ah ok I think just remove it. Let me try to figure out how to do that from here.

**Comment by cafootitt** [comment]
Ok, addressed the comment

**Review by pld — APPROVED** [review]

**Pull request merged**